app/models/suggestion.py

```diff
@@ -69,11 +78,15 @@ def generate_fake_edits(count=20):
             s_read = 0
             s_timestamp = datetime.now(pytz.timezone('US/Eastern'))
             s_contact_name = fake.word()
-            s_contact_email = fake.word() + "@" + fake.word() + ".com"
-            s_contact_number = "123-456-7890"
-            s_edit = Suggestion(resource_id=r_added.id, suggestion_text=s_text,
-                                read=s_read, submission_time=s_timestamp, contact_name=s_contact_name,
-                                contact_email=s_contact_email, contact_phone_number=s_contact_number)
+            s_contact_email = fake.word() + '@' + fake.word() + '.com'
```

Comment:
doesn't show up in www.asylumconnect.org/submit-a-resource/, but we can remove another time